js/build.js

```diff
@@ -9,11 +9,18 @@ Fliplet.Widget.instance('login', function(data) {
     authDefault: 'Verify',
     authProcessing: 'Verifying...',
     sendDefault: 'Send new code',
-    sendProcessing: 'Sending...'
+    sendProcessing: 'Sending...',
+    continuDefault :'Continue',
+    continueProcessing: 'Please wait...'
   };
   _this.$container = $(this);
   _this.data = data;
   _this.pvNameStorage = 'fliplet_login_component';
+  var studioUrls = {
+    'https://api.fliplet.test/': 'http://localhost:8080/',
+    'https://staging.api.fliplet.com/': 'https://staging2.studio.fliplet.com',
+    'https://api.fliplet.com/': 'https://production.studio.fliplet.com'
```

Comment:
the production part can be removed as the main domain is now studio.fliplet.com

Reply:
@squallstar Updated. This is ready for review now.